tests/mk_test.sh

```diff
@@ -155,6 +155,22 @@ function test_mk_build()
   expected_result="make -j$PARALLEL_CORES ARCH=x86_64 "
   assertEquals "($ID)" "$expected_result" "${output}"
 
+  ID=7
+  local count=0
+  cp "$original/tests/samples/.config" .config
+  output=$(mk_build 'TEST_MODE' | head -2) # Remove statistics output
```

Comment:
Remove statistics output? Do we have statistics output here?

Reply:
Yes, `mk_build` always tries to update the statistics folder when run. Most other tests do something similar in this section, maybe we could make it so the statistics part of the code doesn't trigger while in `TEST_MODE`.